pages/news/asteroid-hosttools.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Introducing asteroid-hosttools
+layout: news
+posted: 2023-05-01
+post: true
+thumbnails: asteroid-hosttools
+summary: Asteroid-hosttools make it convenient to work with AsteroidOS smartwatches from a Linux computer.
+author: Timo Könnecke (eLtMosen)
+---
+# Backup and restore has never been easier
+
+With the new software called "[asteroid-hosttools](https://github.com/beroset/asteroid-hosttools)" written by Ed Beroset, it's now easier and faster to work with AsteroidOS smartwatches from a Linux computer. The software comprises three tools: "flashy", "watch-image" and "watch-util".
+
+## Flashy
+Deploys an AsteroidOS image to a supported watch connected via USB easily. Nightly builds are automatically download into the current directory and are used to set up a temporary installation or flash the watch permanently using the real installation.\
+For example, if you have a TicWatch Pro watch (codename catfish) running the stock WearOS image and would like to try out AsteroidOS, `./flashy catfish --nightly --temp` downloads the nightly catfish images and does a temporary install. Later, if you decide to use AsteroidOS permanently, you can install it with `./flashy catfish --nightly`.
+
+## Watch-image
+Allows users to save and restore the original partitions from a watch running WearOS or AsteroidOS. This tool is especially useful for those who would like to use AsteroidOS on their watch for longer time, but would also like to be able to restore the "as shipped" version easily, perhaps to resell the watch.\
+Saving the complete watch into an image file is as convenient as `./watch-image <codename> --save`. Restoring was a complicated process as of yet, involving manually extracting the partitions, figuring out which ones are the correct ones and flash those back individually. Now we can use `./watch-image <codename> --restore` and the watch-image script will handle all of those tasks for us. The `-- dissect` feature extracts the partitions from a watch image for manual inspection.
+
+## Watch-util
+Offers the most features of the three tools.
+- Backup/restore settings for a watch
+- Take a screenshot from your watch
+- Set screen to "always on" using MCE
+- List and set timezones on your watch
+- Set the active wallpaper on your watch
+- Sync time from the connected computer
+- Restart the ceres user
+- Reboot to fastboot
+
+Be sure to head over to the [asteroid-hosttools repository](https://github.com/beroset/asteroid-hosttools) and read about all of the features in detail.\
+They are available on GitHub and are compatible with most Linux distributions.
```

Comment:
The previous link already heads people over to GitHub. Why mention this here as well? Might be better to drop the last line?

Reply:
yeah, it sounds doubled, agreed.

I think i might remove the second sentence but extend the first one into some kind of proper "Conclusion". Which was the purpose of artificially pasting the last sentence.

Reply:
Added a conclusion, plz check.

Reply:
Much better, but can you fix the leading `\`?

![image](https://user-images.githubusercontent.com/7857908/233859665-92d2636f-f705-440f-905f-26cf2fb6a6b5.png)